vlasiator.cpp

```diff
@@ -647,6 +647,9 @@ int main(int argn,char* args[]) {
       );
    }
 
+   // computing coupling of grids (recalculated when load balancing)
```

Comment:
This should not be needed, as we call it in `initializeGrids()` inside `grid.cpp`